# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/phoc/hotfix-examples.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- phoc-v0.57.0/meson.build.vanilla	2026-09-26 22:05:10.208364113 +0300
+++ phoc-v0.57.0/meson.build	2026-09-26 22:05:26.397260886 +0300
@@ -179,7 +179,7 @@
 subdir('tests')
 subdir('tools')
 subdir('doc')
-subdir('examples')
+#subdir('examples')
 
 run_data = configuration_data()
 run_data.set('ABS_BUILDDIR', meson.current_build_dir())
